### DISCUSSION TITLE
Is it possible to run LS-PrePost input file with OpenRadioss Software?

### DISCUSSION THREAD

**Discussion by Majid-MHZ** [OpenRadioss/OpenRadioss#2689] [Q&A]
Hello everyone,

I want to use LS-PrePost software to model my simulation. After that, I want to use OpenRadioss to run the simulation by using the input file exported from LS-PrePost. I would like to know if it's possible to run the LS-PrePost input with OpenRadioss without making any changes. If not, could someone advise me on what modifications need to be made to the input file?

**Comment by SyazwanSamad** [comment]
Yes, you can. Just make sure that all your LS Dyna keywords are supported by OpenRadioss.
Here are the list of supported keywords (might not be the latest due to rapid OpenRadioss development) https://2022.help.altair.com/2022.1/hwsolvers/rad/topics/solvers/rad/ls_dyna_keywords_r.htm

**Reply by Majid-MHZ** [reply]
Dear @SyazwanSamad, 
Thank you for your help.

I want to know how I can run my simulation faster. I have run a powder compaction simulation consisting of 3 parts (box, punch, and particle). The model consists of about 4000 elements and 900 nodes. I started running this simulation last night, but it is still running. I think it will take 855000 seconds until the simulation is done. I want to know if it makes sense for the simulation to take this long. If yes, how can I speed it up?

Best regards,
Majid

**Reply by SyazwanSamad** [reply]
If not mistaken, conversion of *ELEMENT_SPH to OpenRadioss is not yet supported. Therefore, you're running the simulation using LS-Dyna? 
If you're running using LS-Dyna, then maybe you can try the following tips
1) What is your current termination time and the minimum timestep in your model? Rule of thumb is the ratio of termination time/minimum timestep is more than 1e6, then it is better for you to recreate the model, use a coarser mesh/bigger particle. This tip also valid for other type of solvers (OpenRadioss/Abaqus/etc)
2) Are you running SMP or MPP solver? The LS-Dyna MPP version has an issue where if the particles keep moving to another decomposition domain, it will slow down the calculations significantly. The solution is to run using SMP solver or to invoke automatic re-decomposition. The drawback of SMP solver is it is only scalable up to 8 CPU only.
3) Ensure your box and punch is using rigid material (*MAT_RIGID)

Regards
Syazwan

**Comment by Majid-MHZ** [comment]
Dear @SyazwanSamad ,

I didn't use SPH method, I have considered the particle as a solid part. Actually, for the punch and box, I got some errors when defined them as a discrete rigid part. So, I have modeled them as a solid part as well, and defined a rigid body constraint for them. Indeed, I have modeled my simulation by Abaqus software and then imported the Abaqus input file in LS-PrePost software to convert the Abaqus input file to a compatible input file for OpenRadioss. 
Please take a look at the Exported input file from LS-PrePost which I have attached here. I would appreciate it if you could help me in this regard.

Best regards,
Majid
[New folder.zip](https://github.com/OpenRadioss/OpenRadioss/files/15414364/New.folder.zip)

**Comment by SyazwanSamad** [comment]
You have modeled the box and the puncher as solid element, and *MAT_ELASTIC material with steel properties. The solid element has minimum length of ~0.003mm.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/113438036/d467bc7e-4267-4c57-b5ab-f4927d1e2de5)
This cause a very low timestep of ~9E-11s, and termination time is set at 10sec! Therefore, you can calculate how many cycles that LS-Dyna/OpenRadioss need to calculate before it reaches the termination time, which is a lot.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/113438036/c044ce93-e81b-420a-b4ee-0a1ddf70505f).
Changing the box and puncher *MAT_ELASTIC to *MAT_RIGID can help to mitigate the timestep issue, but at the same time, you need to change your *BOUNDARY_XXX card too.
